dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.ts

```diff
@@ -209,4 +209,67 @@ export class TfmDealCancellationRepo {
 
     return { cancelledDealUkefId: dealId, riskExpiredFacilityUkefIds: updatedFacilityUkefIds };
   }
+
+  /**
+   * Submits a scheduled deal cancellation
```

Comment:
Should the function maybe be called `submitDealCancellationScheduledInTheFuture`

I know its wordy, but it's very clear. Otherwise we have this which'd make me wonder why, when looking at this repo alone, why the "scheduled" aspect can't be handled in `submitDealCancellation`.

In fact, could it be handled in `submitDealCancellation` instead?

```
- submitDealCancellation
- submitScheduledDealCancellation
```

Or any other ideas?

Reply:
Good ideas, I've refactored a bit to make this clearer. What I've done: 
- Amend the docstring to make what this function does (and how it differs from `submitDealCancellation`) a lot clearer
- Renamed this function to `scheduleDealCancellation`
So now we have: 
```
- submitDealCancellation
- scheduleDealCancellation
```

I think that renaming makes it clearer that one function is updating the deal to be cancelled immediately whereas the other one will happen in the future (I think 'schedule' now implies 'in the future').

How does that sound?